charts/jenkins/README.md

```diff
@@ -478,6 +478,11 @@ master:
       enabled: true       # was false
 ```
 
+## Prometheus Metrics
+If you have implemented Prometheus Operator, set `master.prometheus.enabled` to `true` to enable Prometheus metrics.  
```

Comment:
```suggestion

If you want to expose Prometheus metrics you need to install the [Jenkins Prometheus Metrics Plugin](https://github.com/jenkinsci/prometheus-plugin).
It will expose an endpoint (default `/prometheus`) with metrics where a Prometheus Server can scrape.

If you have implemented [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator), you can set `master.prometheus.enabled` to `true` to configure a `ServiceMonitor` and `PrometheusRule`.
If you want to further adjust alerting rules you can do so by configuring `master.prometheus.alertingrules`

```